### DISCUSSION TITLE
Takım Görevleri 1- Web sitesi yapımı

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/BilsemPython#81] [Göster ve Söyle]
Takımların birinci görevi, takım arkadaşlarını tanıtan bir web sitesi yapmak. Takım üyeleri gerçek bilgilerini tam olarak girmek zorunda değil, hayali karakterler olabilirsiniz. 

Web sitesini yapmak için ücretsiz github imkanlarından yararlanacaksınız. Her hangi biriniz repositories kısmından kendi adı ve .github.io devamı uzantısı ile bir web sitesi oluşturabilir. (detayları internetten araştırın. örneğin benim sonsuzus.github.io isimli bir repositoriesim var ve otomatik olarak https://sonsuzus.github.io adresinde yayın yapıyor)

Web sitenizi html olarak veya Jekyll kullanarak yapabilirsiniz. (araştırınız)

Süre: 5 Şubat a kadar
Web sitesi yapımı : 60 puan
Birden fazla sayfa ve linklerle bağlanması : +10 puan
Her sayfada ve kişilerin tanıtımı için resim kullanılması: +10 puan
Türkçe dil ve imla kurallarına uyma: +10 puan
Sayfa düzeni ve estetik : +10 puan

Notlar: Takımdaki her üye kendini tanıtan bölümü kendi cümleleri ile olacaktır.
Takıldığınız konuları bu mesajın altına yorum olarak yazabilirsiniz.
Süre bitimine kadar web sitenizin linkini bu sayfa altında paylaşın.

**Comment by AcelyaP** [comment]
Hocam biz grubumuzun adını mı kullanalım yoksa herkes kendi adı ile mu web sitesi oluştursun?

**Reply by sonsuzus** [reply]
Herhangi birinizin github kullanıcı adı ile olabilir. Ayrı ayrı değil tek site yapacaksınız.

**Reply by CinarSatir** [reply]
herkez kendisi site yapsa 
sonrada o siteleri bir araya toplayıp bir site içine atsak olmazmı ?

**Reply by sonsuzus** [reply]
O kısım size kalmış. Github da ortak proje yapma var. Yani hepiniz aynı sayfa üzerinde de çalışabilirsiniz.

**Comment by Ardemin** [comment]
Merhaba ben Arın Demir İnan salı günü 3 ve 4. saatlerde bilişime geliyorum. İstek attığım guruplar dolu olduğu için açıkta kaldım. Boş yeri olan ve beni almak isteyen bir grup var mı?
iyi akşamlar dilerim

**Reply by mervedam** [reply]
Merhaba arın demir phoenix takımında açık var istersen takıma katılabilirsin

**Reply by Ardemin** [reply]
Merhaba Merve takım davetin için çok teşekkürler tabii ki katılmak isterim.Takım için bir telegram ve ya whatsapp grubunuz varsa lütfen beni ekleyin böylece daha kolay bir şekilde iletişime geçebiliriz.
telefon numaram: 530 382 02 05
iyi akşamlar

**Comment by Adaisland** [comment]
Hocam merhaba ödevin teslimi 5 şubatta kadar olduğu için bugün atmalıyız değil mi?Yani ödevin teslimi 00.00'da bitiyor mu?

**Comment by AcelyaP** [comment]
KittyCats websitesi

https://acelyap.github.io/

Hocam bazı kişilere ulaşamadık o yüzden bir kaç kişinin özelliğini giremedik

**Reply by sonsuzus** [reply]
80 Puan

**Reply by sonsuzus** [reply]
Takım kaptanları başlığında sana başvurular var. Onlarla da iletişime geç

**Comment by OmerAkbayrak** [comment]
Black Rhinos Grubu websitesi
https://omerakbayrak.github.io/BlackRhinos_/

**Reply by sonsuzus** [reply]
90 Puan

**Comment by adasue** [comment]
İyi günler Ufuk Hocam.
Monsters takımı sitemizin linki:
https://monstersteamaccount.github.io/index.html

İyi akşamlar

**Reply by sonsuzus** [reply]
100 Puan

**Comment by KayraMertAydin** [comment]
İyi akşamlar Ufuk Hocam.
Takım sitemizin linki:
https://kayramertaydin.github.io/

**Reply by EfeMertIrez** [reply]
Çok güzel olmuş

**Reply by KayraMertAydin** [reply]
teşekkürler

**Reply by sonsuzus** [reply]
90 Puan

**Comment by Furkanyildirm** [comment]
Calcurators takımının linki;
https://furkanyildirm.github.io/Furkanyildirm.github.io

**Reply by sonsuzus** [reply]
90 Puan

**Comment by akgembi12** [comment]
On The Way:
https://akgembi12.github.io

**Reply by sonsuzus** [reply]
90 Puan

**Comment by tolga0102** [comment]
Piton Yılanları takımının web sitesi:
https://tolga0102.github.io

**Reply by sonsuzus** [reply]
80 Puan

**Comment by CinarSatir** [comment]
Rookies takımı web sitesi : 
https://cinarsatir.github.io/rookies.github.io/
sömestr tatilinde bitirmiştim hocam ama 
takım arkadaşlarım fotoraflarını yüklemediği için atmamıştım

**Reply by sonsuzus** [reply]
90 puan
